### PR TITLE
Boxpack

### PR DESCRIPTION
**Pull request by k4gi**
#### Summary

```SUMMARY: Content "Adds new wooden box backpack"```

#### Describe the solution

I present for your consideration, a wooden box you can strap to your back! I copied some stats from the wooden frame and jerrypack.

#### Describe alternatives you've considered

None

#### Testing

I've spawned some items in to make sure the item and recipe are as i expected.

2020-01-06_12:48:06_+1100_screencut

### CODE REVIEW

**Comment by 8street** [comment]
To make a box, you must also use a saw, in my opinion.

**Comment by k4gi** [comment]
> To make a box, you must also use a saw, in my opinion.

quite possibly! but this is not the case for the wooden frame, or the box you can install to a vehicle with one

File: data/json/recipes/armor/storage.json [hunk 1]
```diff
@@ -387,6 +387,18 @@
...
+    "result": "boxpack",
+    "type": "recipe",
+    "category": "CC_ARMOR",
+    "subcategory": "CSC_ARMOR_STORAGE",
+    "skill_used": "fabrication",
+    "difficulty": 2,
+    "time": "6 m",
+    "autolearn": true,
+    "qualities": [ { "id": "HAMMER", "level": 1 } ],
+    "components": [ [ [ "2x4", 5 ] ], [ [ "nail", 20 ] ], [ [ "rope_natural", 1, "LIST" ] ] ]
```
Comment (I-am-Erk) [inline data/json/recipes/armor/storage.json]:
```suggestion
    "components": [ [ [ "2x4", 3 ] ], [ [ "wood_panel", 1 ] ], [ [ "nail", 20 ] ], [ [ "rope_natural", 1, "LIST" ] ] ]
```

File: data/json/recipes/armor/storage.json [hunk 2]
```diff
@@ -387,6 +387,18 @@
...
+  {
+    "result": "boxpack",
+    "type": "recipe",
+    "category": "CC_ARMOR",
+    "subcategory": "CSC_ARMOR_STORAGE",
+    "skill_used": "fabrication",
+    "difficulty": 2,
+    "time": "6 m",
+    "autolearn": true,
+    "qualities": [ { "id": "HAMMER", "level": 1 } ],
```
Comment (I-am-Erk) [inline data/json/recipes/armor/storage.json]:
```suggestion
    "qualities": [ { "id": "HAMMER", "level": 2 }, { "id": "SAW_W", "level": 2 } ], ],
```

File: data/json/recipes/armor/storage.json [hunk 3]
```diff
@@ -387,6 +387,18 @@
...
     "components": [ [ [ "rag", 6 ], [ "leather", 6 ], [ "cordage_short", 3, "LIST" ] ], [ [ "jerrycan", 1 ] ] ]
   },
+  {
+    "result": "boxpack",
+    "type": "recipe",
+    "category": "CC_ARMOR",
+    "subcategory": "CSC_ARMOR_STORAGE",
+    "skill_used": "fabrication",
+    "difficulty": 2,
+    "time": "6 m",
```
Comment (I-am-Erk) [inline data/json/recipes/armor/storage.json]:
```suggestion
    "time": "60 m",
```
There's no way you're making a box from scratch in five minutes.

**Comment by k4gi** [comment]
seems good to me !

**Pull request merged**